proxy/src/main/java/com/velocitypowered/proxy/command/ServerCommand.java

```diff
@@ -49,27 +49,32 @@ public void execute(CommandSource source, String @NonNull [] args) {
       String currentServer = player.getCurrentServer().map(ServerConnection::getServerInfo)
           .map(ServerInfo::getName)
           .orElse("<unknown>");
-      player.sendMessage(TextComponent
-          .of("You are currently connected to " + currentServer + ".", TextColor.YELLOW));
+      player.sendMessage(TranslatableComponent
```

Comment:
For consistency, should we extract all translated components to static final (constant) variables? It would save the duplicate lookups every time these commands get executed.

Reply:
This is not possible for `TranslatableComponent`s with an argument.

Reply:
Take a look at `Translatable`, which can be stored in constant variables and makes the code cleaner.